packages/react-router/package.json

```diff
@@ -29,7 +29,7 @@
     "react-router-dom": "^5.0.1"
   },
   "devDependencies": {
-    "typescript": "~3.2.1"
```

Comment:
do we need this in `devDependencies` when it is in `dependencies` as well?

Reply:
Oh snap, good catch. It was from a bad rebase. I'll check through the rest